core/services/relay/evm/ocr2vrf.go

```diff
@@ -59,11 +60,15 @@ func NewOCR2VRFRelayer(db *sqlx.DB, chain legacyevm.Chain, lggr logger.Logger, e
 }
 
 func (r *ocr2vrfRelayer) NewDKGProvider(rargs commontypes.RelayArgs, pargs commontypes.PluginArgs) (DKGProvider, error) {
+
+	// commontypes.Relay doesn't wire context through
```

Comment:
Let's leave a reference to https://smartcontract-it.atlassian.net/browse/BCF-2887 so we don't miss them